mw_jwt.go

```diff
@@ -284,6 +284,12 @@ func (k *JWTMiddleware) processCentralisedJWT(r *http.Request, token *jwt.Token)
 			log.Error("Could not find a valid policy to apply to this token!")
 			return errors.New("Key not authorized: no matching policy"), http.StatusForbidden
 		}
+		//override session expiry with JWT if it is longer lived
+		if k.Spec.JWTExpiryOverridesPolicy {
+			if int64(claims["exp"].(float64)) > session.Expires {
```

Comment:
I think - because JWT is not generated by Tyk, rather by a 3rd party - that the type assertion here should check validity otherwise the gateway will panic.

Reply:
^ yes, make sense to do right type checking, who knows which garbage can be inside this record.